client/my-sites/hosting/cache-card/index.js

```diff
@@ -8,42 +13,129 @@ import { clearWordPressCache } from 'calypso/state/hosting/actions';
 import getRequest from 'calypso/state/selectors/get-request';
 import { shouldRateLimitAtomicCacheClear } from 'calypso/state/selectors/should-rate-limit-atomic-cache-clear';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { EdgeCacheLoadingPlaceholder } from './edge-cache-loading-placeholder';
+import { useClearEdgeCacheMutation } from './use-clear-edge-cache';
+import { useEdgeCacheQuery } from './use-edge-cache';
+import { useToggleEdgeCacheMutation } from './use-toggle-edge-cache';
 
-const CacheCard = ( {
+const Hr = styled.hr( {
+	marginBottom: '16px',
+	marginTop: '16px',
+	width: '100%',
+	color: '#e0e0e0',
+} );
+
+const EdgeCacheDescription = styled.p( {
+	fontSize: '16px',
+	marginBottom: '16px',
+} );
+
+const EdgeCacheNotice = styled.p( {
+	color: '#646970',
+	marginTop: '18px',
+} );
+
+const CardBody = styled.div( {
+	display: 'flex',
+	flexDirection: 'column',
+} );
+
+const ToggleContainer = styled.div( {
+	fontSize: '14px',
+	label: {
+		fontSize: '14px',
+	},
+} );
+
+const CacheToggle = styled( ToggleControl )( {
+	marginBottom: '0px !important',
+} );
```

Comment:
I would prefer to avoid using `!important`. And testing it I couldn't see any visual effect added by this CSS style.


https://github.com/Automattic/wp-calypso/assets/779993/00322c07-d654-4038-8211-b3425e92b7af